scripts/drone/steps/lib.star

```diff
@@ -327,9 +335,9 @@ def e2e_tests_artifacts():
         "environment": {
             "GCP_GRAFANA_UPLOAD_ARTIFACTS_KEY": from_secret(gcp_upload_artifacts_key),
             "E2E_TEST_ARTIFACTS_BUCKET": "releng-pipeline-artifacts-dev",
-            "GITHUB_TOKEN": from_secret("github_token"),
         },
         "commands": [
+            "GITHUB_TOKEN=$(cat /github-app/token)",
```

Comment:
```suggestion
            "export GITHUB_TOKEN=$(cat /github-app/token)",
```

We had issues in other pipelines in enterprise where we forgot `export`. I think it's because these aren't running in a single bash script, but rather in `sh`. Maybe. something like that.